test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -362,7 +362,53 @@ static public void testWithFieldOnNonExistentClass() throws Throwable {
 			Assert.fail("should throw error. Class does not exist");
 		} catch (NoClassDefFoundError e) {}
 	}
-	
+
+	/*
+	 * Test ifacmp on value class
+	 * 
+	 * class TestIfacmpOnValueClass {
+	 *  
+	 * }
```

Comment:
I think you need to test the following
1) VT with VT ==
2) VT with VT !=
3) VT with Ref ==
4) VT with Ref !=
5) Ref with VT ==
6) Ref with VT !=
7) Ref with Ref ==
8) Ref with Ref !=

Reply:
I write these tests as conditional statements in this function, do you mean I have to spilt it into functions respectively?